service/src/main/java/tds/session/web/endpoints/HealthController.java

```diff
@@ -0,0 +1,18 @@
+package tds.session.web.endpoints;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/session")
```

Comment:
Does this mean the URL for this will be something like http://session.sbac.org/session/endpoint?  Is the /session needed or will it be taken care of by the fact that we are hitting the session service?  I haven't really thought it thru so I'm probably missing something.